src/ec/packages/ec-component-page-banner/.npmignore

```diff
@@ -0,0 +1,5 @@
+__snapshots__
+docs
+*.story.js
+*.test.js
+**/*.md
```

Comment:
actually we want the main README.md file to be published on npmjs.com, thus we don't want to ignore it ;)

Reply:
ok, but we will have to update `.npmignore` in other compoonents, as it is the same